I'm trying to run "[Using conda to provide all dependencies for the Sage library (experimental)](https://github.com/mkoeppe/sage/blob/355b39cce59ad9e5cb3e5c8b16915410e83f8591/src/doc/en/installation/conda.rst#using-conda-to-provide-all-dependencies-for-the-sage-library-experimental)" on this branch, and get
```
$ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable ./src
...
  adding 'sage_conf-9.8.dist-info/top_level.txt'
  adding 'sage_conf-9.8.dist-info/RECORD'
  removing build/bdist.linux-x86_64/wheel
  Building wheel for sage-conf (pyproject.toml) ... done
  Created wheel for sage-conf: filename=sage_conf-9.8-py3-none-any.whl size=5929 sha256=2452cdf180ceebae3ac425dc4be670ed835ba9e14dabadcca092846ec7d3003b
  Stored in directory: /home/scratch/dimpase/.cache/pip/wheels/42/8d/24/ee2834b08081c68f0473dced1cfa0c801641dffdf6648cb811
Successfully built sage-conf
Failed to build sagemath-standard
ERROR: Could not build wheels for sagemath-standard, which is required to install pyproject.toml-based projects
Exception information:
Traceback (most recent call last):
  File "/home/scratch/scratch2/dimpase/mambaforge/envs/sage-dev/lib/python3.11/site-packages/pip/_internal/cli/base_command.py", line 180, in exc_logging_wrapper
    status = run_func(*args)
             ^^^^^^^^^^^^^^^
  File "/home/scratch/scratch2/dimpase/mambaforge/envs/sage-dev/lib/python3.11/site-packages/pip/_internal/cli/req_command.py", line 248, in wrapper
    return func(self, options, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/scratch/scratch2/dimpase/mambaforge/envs/sage-dev/lib/python3.11/site-packages/pip/_internal/commands/install.py", line 429, in run
    raise InstallationError(
pip._internal.exceptions.InstallationError: Could not build wheels for sagemath-standard, which is required to install pyproject.toml-based projects
Remote version of pip: 23.2.1
Local version of pip:  23.2.1
Was pip installed by pip? False
```
Any idea what it could be?

I don't know whether it's a regression or not, though.

--- a/.github/workflows/ci-conda.yml
+++ b/.github/workflows/ci-conda.yml
@@ -92,7 +92,7 @@ jobs:
         run: |
           # Use --no-deps and pip check below to verify that all necessary dependencies are installed via conda.
           pip install --no-build-isolation --no-deps -v -v -e ./pkgs/sage-conf ./pkgs/sage-setup
-          pip install --no-build-isolation --no-deps -v -v -e ./src
+          pip install --no-build-isolation --no-deps --config-settings editable_mode=compat -v -v -e ./src
         env:
           SAGE_NUM_THREADS: 2
         

--- a/build/pkgs/setuptools/distros/conda.txt
+++ b/build/pkgs/setuptools/distros/conda.txt
@@ -1,4 +1,1 @@
-# Set this bound until https://github.com/sagemath/sage/issues/34209 adds support for PEP660 editable builds
-# By setting this version bound, we avoid having to include the following in our installation instructions.
-#   export SETUPTOOLS_ENABLE_FEATURES=legacy-editable
-"setuptools<64"
+setuptools

--- a/build/pkgs/setuptools_wheel/distros
+++ b/build/pkgs/setuptools_wheel/distros
@@ -1,0 +1,1 @@
+../setuptools/distros

--- a/build/pkgs/setuptools_wheel/distros/repology.txt
+++ b/build/pkgs/setuptools_wheel/distros/repology.txt
@@ -1,1 +1,0 @@
-python:setuptools

--- a/src/doc/en/installation/conda.rst
+++ b/src/doc/en/installation/conda.rst
@@ -158,7 +158,7 @@ suffices to restart Sage.
 
 After editing any Cython files, rebuild the Sage library using::
 
-  $ pip install --no-build-isolation -v -v --editable src
+  $ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable src
 
 In order to update the conda environment later, you can run::
 

--- a/src/doc/en/installation/conda.rst
+++ b/src/doc/en/installation/conda.rst
@@ -134,8 +134,9 @@ Here we assume that you are using a git checkout.
     installed. You can use the additional option ``python=3.9`` in the above
     ``env create`` command to select another Python version (here 3.9).
 
-  - Install the build prerequisites and the Sage library::
+  - Bootstrap the source tree and install the build prerequisites and the Sage library::
 
+      $ ./bootstrap
       $ pip install --no-build-isolation -v -v --editable ./pkgs/sage-conf_conda ./pkgs/sage-setup
       $ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable ./src
 

